go.mod

```diff
@@ -14,8 +14,8 @@ require (
 	github.com/kyma-project/kyma/common/logging v0.0.0-20231220133808-e2c22b79a28b
 	github.com/kyma-project/nats-manager v1.0.3-0.20231219150808-13159cfea47c
 	github.com/mitchellh/hashstructure/v2 v2.0.2
-	github.com/nats-io/nats-server/v2 v2.10.10
-	github.com/nats-io/nats.go v1.33.0
+	github.com/nats-io/nats-server/v2 v2.10.11
```

Comment:
are the NATS libs compatible with the NATS server version we deploy?

Reply:
The e2e test that utilise NATS server are green.